scripts/create-kubestellar-demo-env.sh

```
@@ -152,58 +173,80 @@ echo -e "\nCreating two $k8s_platform clusters to serve as example WECs"
+    # This is a synchronous script call
+    curl -s \
+         https://raw.githubusercontent.com/kubestellar/kubestellar/v${kubestellar_version}/scripts/create-kind-cluster-with-SSL-passthrough.sh \
+         | bash -s -- --name kubeflex --nosetcontext
 else
+    # Ensure port 9443 is free before binding
+    wait_for_port_free 9443 120
+    # Also a synchronous command
     k3d cluster create -p "9443:443@loadbalancer" --k3s-arg "--disable=traefik@server:*" kubeflex
     sleep 15
-    helm install ingress-nginx ingress-nginx --set "controller.extraArgs.enable-ssl-passthrough=" --repo https://kubernetes.github.io/ingress-nginx --version 4.11.3 --namespace ingress-nginx --create-namespace
+    helm install ingress-nginx ingress-nginx \
+        --set "controller.extraArgs.enable-ssl-passthrough=" \
+        --repo https://kubernetes.github.io/ingress-nginx \
+        --version 4.11.3 \
+        --namespace ingress-nginx \
+        --create-namespace
 fi
 echo -e "\033[33m✔\033[0m Completed KubeFlex cluster with SSL Passthrough"
 
-wait
 kubectl config use-context $k8s_platform-kubeflex
 
 some_failed=false
 for cluster in "${clusters[@]}"; do
     if ! [ -f "${cluster_log_dir}/${cluster}.success" ]; then
-	echo -e "\033[0;31mX\033[0m Creation of cluster $cluster failed!" >&2
-	cat "${cluster_log_dir}/${cluster}.log" >&2
-	some_failed=true
-	continue
+        echo -e "\033[0;31mX\033[0m Creation of cluster $cluster failed!" >&2
+        cat "${cluster_log_dir}/${cluster}.log" >&2
+        some_failed=true
+        continue
     fi
     echo -e "\033[33m✔\033[0m Cluster $cluster was successfully created"
-    kubectl config rename-context "${k8s_platform}-${cluster}" "${cluster}" >/dev/null 2>&1
+    kubectl config rename-context "${k8s_platform}-${cluster}" "${cluster}" >/dev/null 2>&1 || true
```

Why add `|| true` here? It masks failures.